clients/fossid-webapp/src/main/kotlin/Extensions.kt

```diff
@@ -30,9 +30,10 @@ private const val SCAN_GROUP = "scans"
 private const val PROJECT_GROUP = "projects"
 
 /**
- * This function verifies that a request to FossID server was successful
- * @param operation the label of the operation, for logging
- * @param withDataCheck if true, the payload will be checked for existence
+ * This function verifies that a request to FossID server was successful.
+ * [operation] will be used as label of the operation for logging
+ *
+ * If [withDataCheck] is true, the payload will be checked for existence
```

Comment:
Should be something like:

Verify that a request for the given [operation] was successful. If [withDataCheck] is true, also the payload data is checked, otherwise that check is skipped.

Reply:
In general, you can omit the mention of "FossID" / "FossID server" from the docs, as this is all about FossID.

Reply:
Ok I did the changes.
Nevertheless I added a sentence to describe the `operation` param otherwise this would be a mistery prameter for the caller.